tests/build/const.go

```diff
@@ -23,6 +23,9 @@ const (
 	helloWorldComponentDefaultBranch     = "default"
 	helloWorldComponentRevision          = "d2d03e69de912e3827c29b4c5b71ffe8bcb5dad8"
 
+	helloWorldComponentGitSourceCloneRepoName = "devfile-sample-hello-world-clone"
```

Comment:
should we revert these repo naming changes?

Reply:
Probably some leftover.. was testing with cloning the actual repo.. let me revert it back

Reply:
Changed now.

Reply:
Reverted back the change, could not remember earlier why it is used, it is needed to used in [this test](https://github.com/konflux-ci/e2e-tests/pull/1309/files#diff-938ddf0d569d68c3df496d8b4065614ff56cc303c33555755621cffcec97ef74R1170) to avoid conflict with sample repo used in [this test](https://github.com/konflux-ci/e2e-tests/pull/1309/files#diff-938ddf0d569d68c3df496d8b4065614ff56cc303c33555755621cffcec97ef74R43)